contrib/coreos/user-data.example

```diff
@@ -133,3 +134,14 @@ write_files:
       TOOLBOX_DOCKER_IMAGE=ubuntu-debootstrap
       TOOLBOX_DOCKER_TAG=14.04
       TOOLBOX_USER=root
+  - path: /etc/environment_proxy
+    owner: core
+    content: |
+      HTTP_PROXY=
+      HTTPS_PROXY=
+      ALL_PROXY=
+      NO_PROXY=
+      http_proxy=
+      https_proxy=
+      all_proxy=
+      no_proxy=
```

Comment:
Why both uppercase and lowercase? I'm familiar with the lowercase form for many unix commands, but not the uppercase form as a shell environment variable (outside of CGI which is a different use context and scope).